Dockerfile

```diff
@@ -1,7 +1,29 @@
 FROM python:3.7-slim-buster
 ENV PYTHONUNBUFFERED 1
```

Comment:
Would be nice to have the environment vars that the project accept, declared here.

Reply:
Those would be: CELERY_LOGGER_BROKER and CELERY_LOGGER_LOG_FILE

Reply:
Done!